source/js/viewer-core.js

```diff
@@ -404,7 +404,8 @@ function ViewerCore(element, options, publicInstance)
             var options = {
                 viewport: viewerState.viewport,
                 outerElement: viewerState.outerElement,
-                innerElement: viewerState.innerElement
+                innerElement: viewerState.innerElement,
+                enableCrossOrigin: settings.options.enableCrossOrigin
```

Comment:
my first try was to add `enableCrossOrigin` to the viewer state but it seems strange because it was just set from the diva root options and since the viewer has access to root options just use that